extensions/2.0/Vendor/EXT_pbr_attributes/README.md

```diff
@@ -0,0 +1,223 @@
+# EXT_pbr_attributes
+
+## Contributors and Acknowledgements
+
+* Florian Hoenig, Unbound Technologies, [@rianflo](http://twitter.com/rianflo)
+* Michael Saenger, Unbound Technologies, [@abductee_org](http://twitter.com/abductee_org)
+* Alex Evans, Mediamolecule, [@mmalex](https://twitter.com/mmalex)
+* Gary Hsu, Microsoft, [@bghgary](https://twitter.com/bghgary)
+* Warren Moore, metalbyexample.com, [@warrenm](https://twitter.com/warrenm)
+* David Farrell, Oculus, [@Nosferalatu](https://twitter.com/Nosferalatu)
+* Don McCurdy, Google, [@donrmccurdy](https://twitter.com/donrmccurdy)
+* Ocean Quigley, Facebook, [@oceanquigley](https://twitter.com/oceanquigley)
+* Scott Nagy, Microsoft, [@visageofscott](https://twitter.com/visageofscott)
+* Milan Bulat, Foundry, [@speedy_milan](https://twitter.com/speedy_milan)
+
+## Status
+
+Draft
+
+TODO: 
+* Update sample file
+* Update schema file
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines the metalness-roughness material model from Physically-Based Rendering (PBR) per vertex attribute.
+These additional attribute semantics referenced in the material definition.
+
+### Example
+![\[Comparison\]](Figures/vertex_metal_rough_comparison.png)
+
+Left: per-vertex albedo only. Right: per-vertex albedo attributes extended with metalness-roughness
+
+[painted_sphere.glb](examples/painted_sphere.glb)
+
+## glTF Schema Updates
+
+### Examples 
+```C
+{
+    "asset": {
+        "generator": "Unbound/PlayEngine",
+        "version": "2.0"
+    },
+    "scene": 0,
+    "extensionsUsed": [
+        "EXT_pbr_attributes"
+    ],
+    "materials": [
+        {
+            "name": "material_basic_def",
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [1, 1, 1, 1],
+                "metallicFactor" : 1,
+                "roughnessFactor": 0
+            },
+            "extensions" : {
+                "EXT_pbr_attributes" : {
+                    "roughnessAttrib" : "_ROUGHNESS",
+                    "metallicAttrib"  : "_METALLIC",
+                }
+            }
+        },
+        {
+            "name": "material_0",
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [1, 1, 1, 1],
+                "metallicFactor" : 1,
+                "roughnessFactor": 0
+            },
+            "extensions" : {
+                "EXT_pbr_attributes" : {
+                    "baseColorSpace"  : "sRGB",  
+                    "baseColorAttrib" : "_ALBEDO",
+                    "roughnessAttrib" : "_ROUGHNESS",
+                    "metallicAttrib"  : "_METALLIC",
+                }
+            }
+        },

```

Comment:
The schema does not match the keys present in this example. The schema only contains `"baseColorAttribSpace"`, which is none of these.